### DISCUSSION TITLE
How to attach a custom CloudFront Function to construct?

### DISCUSSION THREAD

**Discussion by victorronnow** [getlift/lift#409] [Constructs]
Hi!

Is there currently a way to attach a CloudFront Function to a static website construct?

I'd like to rewrite the URI to strip out the .html extension in the address bar.

I'm able to create it manually inside the AWS CloudFront dashboard, but is there any way to write this in serverless.yml config?

Thank you!

function handler(event) {
    var request = event.request;
    var uri = request.uri;

    if (!uri.includes('.') && !uri.endsWith('/')) {
        request.uri = `${uri}.html`;
    }

    return request;
}

**Comment by ilyaLibin** [comment]
Hi, have you figured that out? I guess you are using next.js to build the static website?
